pom.xml

```diff
@@ -1818,6 +1818,27 @@
     </build>
 
     <profiles>
+        <profile>
```

Comment:
Please move this to the Phoenix POMs. We don’t want to enable this for everything. It can allow regressions.